kura/org.eclipse.kura.core.system/src/main/java/org/eclipse/kura/core/system/SystemServiceImpl.java

```diff
@@ -1483,5 +1483,15 @@ public boolean isLegacyBluetoothBeaconScan() {
 
         return false;
     }
+    
+    @Override
+    public boolean isLegacyPPPLoggingEnabled() {
+        final Optional<String> override = getProperty(KEY_LEGACY_PPP_LOGGING);
+        if (override.isPresent()) {
+            return Boolean.parseBoolean(override.get());
+        }
+
+        return false;
```

Comment:
I thought about that. But considering that our targets from now on are systemd systems where the PPP daemon can directly write to the journal, I thought that maybe this property is better to be false by default and for legacy systems it can be set to true. 
But I'm open to change the default if we believe otherwise.

Reply:
So, the default value is false (a.k.a. log in the journal) and in our profiles we override it and set to true (a.k.a. log in a separate file). It makes sense.